integration/ledger/reset_rollback_test.go

```diff
@@ -481,3 +544,15 @@ func (th *testHelper) assertPausedChannel(peer *nwo.Peer) {
 	Eventually(sess, th.EventuallyTimeout).Should(gexec.Exit(1))
 	Expect(sess.Err).To(gbytes.Say("Invalid chain ID"))
 }
+
+func (th *testHelper) assertUnjoinedChannel(peer *nwo.Peer) {
+	sess, err := th.PeerUserSession(peer, "User1", commands.ChannelInfo{
+		ChannelID: th.channelID,
+	})
+	Expect(err).NotTo(HaveOccurred())
+	Eventually(sess, th.EventuallyTimeout).Should(gexec.Exit(1))
+	Expect(sess.Err).To(gbytes.Say("Invalid chain ID"))
+
+	channelLedgerDir := filepath.Join(th.Network.PeerLedgerDir(peer), "chains/chains", th.channelID)
+	Expect(channelLedgerDir).NotTo(BeADirectory())
```

Comment:
While it does not harm to check for lower levels such as this, it's better to execute a channel specific command to and expect to receive an error.

Reply:
Agree.  At least in this case it was a bit annoying to have the unjoin halt immediately while the peer is running.  (There's at least a little coverage for the error case with the "double unjoin.")  This is a small motivating case but it does seem like ALL of the `peer` commands should be executed against a running instance - in the case of cloud--native fabric deployments, it's a challenge to run things "next to" where the peer would be running.